src/Doctrine/PlatenumDoctrineType.php

```diff
@@ -101,19 +101,30 @@ public function getName(): string
         return $this->platenumAlias;
     }
 
-    public function getSQLDeclaration(array $declaration, AbstractPlatform $platform): string
+    /**
+     * @psalm-suppress ParamNameMismatch
+     */
+    public function getSQLDeclaration(array $column, AbstractPlatform $platform): string
     {
-        return ($this->platenumSql)($declaration, $platform);
+        return ($this->platenumSql)($column, $platform);
     }
 
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
+        if (null === $value || '' === $value) {
+            return null;
+        }
+
         /** @psalm-suppress MixedMethodCall */
         return ($this->platenumCallback)($value->getValue());
     }
 
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
+        if (null === $value || '' === $value) {
```

Comment:
please remove the empty string condition, only `null` should be allowed here

Reply:
Done